java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -1083,51 +1086,13 @@ CompletableFuture<Object[]> bzmpop(
             String[] keys, ScoreFilter modifier, double timeout, long count);
 
     /**
-     * Returns the union of members from sorted sets specified by the given <code>keysOrWeightedKeys
-     * </code>.<br>
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.<br>
      * To get the elements with their scores, see {@link #zunionWithScores}.
      *
-     * @apiNote When in cluster mode, all keys in <code>keysOrWeightedKeys</code> must map to the same
-     *     hash slot.
-     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
-     * @param keysOrWeightedKeys The keys of the sorted sets with possible formats:
-     *     <ul>
-     *       <li>Use {@link KeyArray} for keys only.
-     *       <li>Use {@link WeightedKeys} for weighted keys with score multipliers.
-     *     </ul>
-     *
-     * @param aggregate Specifies the aggregation strategy to apply when combining the scores of
-     *     elements.
-     * @return The resulting sorted set from the union.
-     * @example
-     *     <pre>{@code
-     * KeyArray keyArray = new KeyArray(new String[] {"mySortedSet1", "mySortedSet2"});
-     * String[] payload = client.zunion(keyArray, Aggregate.MAX).get()
-     * assert payload.equals(new String[] {"elem1", "elem2", "elem3"});
-     *
-     * WeightedKeys weightedKeys = new WeightedKeys(List.of(Pair.of("mySortedSet1", 2.0), Pair.of("mySortedSet2", 2.0)));
-     * String[] payload = client.zunion(weightedKeys, Aggregate.MAX).get()
-     * assert payload.equals(new String[] {"elem1", "elem2", "elem3"});
-     * }</pre>
-     */
-    CompletableFuture<String[]> zunion(KeysOrWeightedKeys keysOrWeightedKeys, Aggregate aggregate);
```

Comment:
why are you removing the ZUNION and ZINTER options with keys and aggregate?

Reply:
> Aggregate is noop

Our API should match Redis.  Let's keep it until Redis decides to raise an error or something.

Reply:
I think redis docs should be updated to
```
ZUNION numkeys key [key ...] [[WEIGHTS weight [weight ...]]
  [AGGREGATE <SUM | MIN | MAX>] WITHSCORES]
```
Yes, you can submit `WEIGHTS` and/or `AGGREGATE` to the command, but it does not change the output without `WITHSCORES`. I guess it increases execution time though.

Reply:
@shohamazon confirmed that we can remove those.
I'm glad to continue discussion on upstream.